src/components/mdx/callout.js

```diff
@@ -1,13 +1,9 @@
-import React from 'react'
-
-const container = ''
-
-export default function Callout({children, className}) {
+export default function Callout({children, text, className}) {
   return (
     <div
-      className={`mx-auto my-3 w-full dark:bg-gray-800 bg-blue-100 border-l-4 overflow-hidden border-blue-900 dark:text-blue-200 text-blue-600 sm:px-8 px-4 py-3 rounded-sm`}
+      className={`p-5 leading-relaxed mx-auto my-3 w-full dark:bg-gray-800 bg-blue-100 border-l-4 border-blue-900 sm:space-x-5 space-x-0 sm:space-y-0 space-y-5 items-center text-left shadow-sm rounded-lg overflow-hidden lg:text-lg sm:text-md ${className}`}
```

Comment:
using `className` like this without making it an empty string as a fallback will make it return `undefined` by default.